scripts/chroot-build

```diff
@@ -22,6 +22,7 @@ function umount_system_fs ()
 mount_system_fs &&
 
 mkdir $ISO_TARGET/run/lock &&
+mkdir $ISO_TARGET/run/lock/lunar &&
```

Comment:
You didn't reply to https://github.com/lunar-linux/lunar-iso/pull/29#issuecomment-321242744 .
Where is this needed?